geps/gep-713.md

```diff
@@ -525,13 +783,248 @@ type RouteRule struct {
 ### Disadvantages
 * May be difficult to understand which policies apply to a request
 
+## Examples
+
+This section provides some examples of various types of Policy objects, and how
+merging, `defaults`, `overrides`, and other interactions work.
+
+### Direct Policy Attachment
+
+The following Policy sets the minimum TLS version required on a Gateway Listener:
+```yaml
+apiVersion: networking.example.io/v1alpha1
+kind: TLSMinimumVersionPolicy
+metadata:
+  name: minimum12
+  namespace: default
+spec:
+  minimumTLSVersion: 1.2
+  targetRef:
+    name: internet
+    group: gateway.networking.k8s.io
+    kind: Gateway
+```
+
+Note that because there is no version controlling the minimum TLS version in the
+Gateway `spec`, this is an example of a non-field Policy.
+
+### Hierarchical Policy Attachment
+
+It also could be useful to be able to _default_ the `minimumTLSVersion` setting
+across multiple Gateways.
+
+This version of the above Policy allows this:
+```yaml
+apiVersion: networking.example.io/v1alpha1
+kind: TLSMinimumVersionPolicy
+metadata:
+  name: minimum12
+  namespace: default
+spec:
+  defaults:
+    minimumTLSVersion: 1.2
+  targetRef:
+    name: default
```

Comment:
The use of "default" as the namespace name here feels confusing, maybe some other more generic term would be helpful so there's no chance that someone thinks that setting "default" as a name has some kind of special meaning.